lib/rules/at-rule-no-deprecated/README.md

```diff
@@ -0,0 +1,70 @@
+# at-rule-no-deprecated
+
+Disallow deprecated at-rules.
+
+<!-- prettier-ignore -->
+```css
+    @document url(http://www.w3.org/);
```

Comment:
You should probably pick another one as an example.
i.e. AFAIK non-vendor-prefixed `@document` was never supported by any browser

ref: https://developer.mozilla.org/en-US/docs/Web/CSS/@document#browser_compatibility